### PR TITLE
fix: Use actual repository url for maester dependency

### PR DESCRIPTION
**Pull request by remotezygote**
The relative file path in use currently breaks outside of straight helm usage, for example when using this chart with `garden`. This also adds `*.txt` to the `.helmignore` files in each of `hydra` and `oathkeeper` as those files were being "built" by helm when testing locally.

## Related issue(s)

https://github.com/ory/k8s/issues/105#issuecomment-952267328

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works. (I don't know how to test this)
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by Demonsthere** [comment]
I took the liberty of doing some housekeeping for this PR ;) It still seams the CLA is unhappy as https://github.com/ory/k8s/pull/351/commits/4637bf3c3fa78425b79d092b2365425ebff58e02 seems to be signed with a different email. Could you try following https://github.com/ory/k8s/pull/214#issuecomment-756615018 to fix that?

File: helm/charts/hydra/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ dependencies:
     version: 0.20.1
     condition: maester.enabled
     alias: hydra-maester
-    repository: file://../hydra-maester
+    repository: https://k8s.ory.sh/helm/charts
```
Comment (Demonsthere) [inline helm/charts/hydra/Chart.yaml:29]: I like this approach, clean and simple :) 
One thing thou, I would like to keep the second, file url as a comment. It is helpful to have it for local development (f.e if you want to modify the maester, and test that for hydra/oathkeeper)

Comment (remotezygote) [reply helm/charts/hydra/Chart.yaml:29]:
```suggestion
    # repository: file://../hydra-maester
    repository: https://k8s.ory.sh/helm/charts
```

File: helm/charts/oathkeeper/Chart.yaml [hunk 2]
```diff
@@ -27,4 +27,4 @@ dependencies:
     version: 0.20.1
     condition: maester.enabled
     alias: oathkeeper-maester
-    repository: file://../oathkeeper-maester
+    repository: https://k8s.ory.sh/helm/charts
```
Comment (Demonsthere) [inline helm/charts/oathkeeper/Chart.yaml:30]:
```suggestion
    # repository: file://../oathkeeper-maester
    repository: https://k8s.ory.sh/helm/charts
```

**Review by Demonsthere — APPROVED** [review]

**Comment by remotezygote** [comment]
Hm, the release where this is supposed to be seems to have changed these lines at some point?

https://github.com/ory/k8s/blob/v0.20.2/helm/charts/hydra/Chart.yaml#L29-L30

**Comment by Demonsthere** [comment]
Hi @remotezygote yes, I had to revert it for now, as it is a breaking change to the current release process (didn't noticed that beforehand :/). As we release all charts at once, during the release 0.20.2 of oathkeeper, helm tries to download the release 0.20.2 of maester, which is not yet available. This can be seen [here](https://app.circleci.com/pipelines/github/ory/k8s/1221/workflows/a9bb1ffa-60de-4aac-aeed-4b1ad5f4b271/jobs/2752/parallel-runs/0/steps/0-102). 
I'm sorry. but we cannot use the repo url, unless we modify the release pipeline first, to for example work in sequential and upload the charts one after another, or separate the process to be per-chart 😞

**Comment by remotezygote** [comment]
Yeah, was trying to use it with `garden` and it is unworkable in its current state. My only solution is to clone the charts locally in my repo which I don't want to do for the obvious maintenance reasons. I decided to go another route, so won't be pushing this forward more. I would highly recommend doing this, though - people who are using `garden` or similar tools are gonna get pretty frustrated with it and likely give up (kinda what I did).

**Comment by remotezygote** [comment]
Oh! And disabling maester does nothing to disable installing it, which is the issue here. It can't install if its directory structure doesn't match yours. That's not how it's supposed to work. You cannot install the `hydra` or `oathkeeper` charts _at all_ in `garden` because of this.

**Pull request merged**